api_fhir_r4/tests/test/test_claimAdminPractitioner.json

```diff
@@ -0,0 +1,52 @@
+{
+   "resourceType":"Practitioner",
+   "birthDate":"1990-03-24",
+   "identifier":[
+      {
+         "type":{
+            "coding":[
+               {
+                  "code":"FILL",
```

Comment:
Why we have FILL here? We change the types to Code or UUID, and specific usage for DBID.

Reply:
@dragos-dobre I forgot to change this. I will replace this into 'Code' etc. That 'FILL' was before our development in E0 project.